<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gyrflacon Sensor System Professional</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>170</y>
      <width>201</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QDial" name="dialTimeScale">
     <property name="minimum">
      <number>60</number>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDial" name="dialTimeStart">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDial" name="dial_3"/>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>RESULT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>530</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>UD Digi Kyokasho NK-R</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CPU Temperature</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBarChipTemp">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::BottomToTop</enum>
       </property>
       <property name="format">
        <string>%pâ„ƒ</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>UD Digi Kyokasho NK-R</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>COM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberCOM">
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="digitCount">
        <number>6</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>UD Digi Kyokasho NK-R</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BAUDRATE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberBaudrate">
       <property name="digitCount">
        <number>6</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButtonOpenCOM">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>OPENCOM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCloseCom">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>CLOSECOM</string>
     </property>
    </widget>
   </widget>
   <widget class="QChartView" name="ChartSensor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimeScale">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAnalyse">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ANALYSE CHART</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionContast_us"/>
    <addaction name="actionGyrflaconSensorSystem"/>
    <addaction name="separator"/>
    <addaction name="actionLicenseCode"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCOM_Setting"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionContast_us">
   <property name="text">
    <string>Contast us</string>
   </property>
  </action>
  <action name="actionGyrflaconSensorSystem">
   <property name="text">
    <string>About Gyrflacon System</string>
   </property>
  </action>
  <action name="actionCOM_Setting">
   <property name="text">
    <string>COM Setting</string>
   </property>
  </action>
  <action name="actionLicenseCode">
   <property name="text">
    <string>ActiveSoftware</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>PyQt5.QtChart</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionContast_us</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenContactus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGyrflaconSensorSystem</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenSysinfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLicenseCode</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenActiveSoftware()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCOM_Setting</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenCOMSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOpenCOM</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ConnectCOM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCloseCom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DisconnectCOM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>947</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialTimeScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetTimeScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>1071</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialTimeStart</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetTimeStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAnalyse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AnalyseChart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenContactus()</slot>
  <slot>OpenSysinfo()</slot>
  <slot>OpenActiveSoftware()</slot>
  <slot>OpenCOMSetting()</slot>
  <slot>ConnectCOM()</slot>
  <slot>DisconnectCOM()</slot>
  <slot>SetTimeScale()</slot>
  <slot>SetTimeStart()</slot>
  <slot>AnalyseChart()</slot>
 </slots>
</ui>
